packages/babel-traverse/src/scope/index.js

```diff
@@ -140,7 +140,7 @@ const collectorVisitor = {
   },
 
   UpdateExpression(path, state) {
-    state.constantViolations.push(path.get("argument"));
+    state.constantViolations.push(path);
   },
 
   UnaryExpression(path, state) {
```

Comment:
As far as I know the delete operator cannot violate the const property ( this would be missing from the current check constants plugin). I am not sure why it is added and therefore did not change it.

Why is a delete node added to constantViolations?